src/screens/login/Login.tsx

```diff
@@ -0,0 +1,76 @@
+import {Pressable, StyleSheet, Text, TextInput, View} from 'react-native';
+import React from 'react';
+import {useLogin} from './useLogin';
+import {LoginParam} from './types';
+import {StackNavigationProp} from '@react-navigation/stack';
+import {CONFIRM_TITLE, LOGIN_TITLE} from './constants';
+
+export default function Login({
+  navigation,
+}: {
+  navigation: StackNavigationProp<LoginParam, 'Login'>;
+}) {
+  const {isValidUser, handleUsernameChange, handlePasswordChange} = useLogin({
+    navigation,
+  });
+
+  return (
+    <View style={style.screenContainer}>
+      <Text style={style.header}>{LOGIN_TITLE}</Text>
+      <View style={style.container}>
+        <TextInput
+          style={style.inputContainer}
+          onChangeText={handleUsernameChange}
+        />
+        <TextInput
+          style={style.inputContainer}
+          secureTextEntry={true}
+          onChangeText={handlePasswordChange}
+        />
+        <Pressable style={style.confirmButton} onPress={isValidUser}>
+          <Text style={style.confirmText}>{CONFIRM_TITLE}</Text>
+        </Pressable>
+      </View>
+    </View>
+  );
+}
+
+const style = StyleSheet.create({
+  screenContainer: {
+    flex: 1,
+    backgroundColor: '#F3F3E0',
```

Comment:
Create a folder called theme inside src and there create a file called colors and put all the colors there.